app.js

```diff
@@ -155,31 +163,33 @@ function LeftButton(route, navigator, index, navState) {
       if (index === 1) {
         backTitle = 'Home'
       }
-      return (
-        <TouchableOpacity
-          style={styles.backButton}
-          onPress={() => navigator.pop()}
-        >
-          <Icon style={styles.backButtonIcon} name='ios-arrow-back' />
-          <Text style={styles.backButtonText}>{backTitle}</Text>
-        </TouchableOpacity>
-      )
+      if (Platform.OS === 'android') {
+        return (
+          <TouchableOpacity
+            style={styles.backButton}
+            onPress={() => navigator.pop()}
+          >
+              <Icon style={styles.backButtonIcon} name='md-arrow-back' />
```

Comment:
Weirder: It renders the icon on my device, but it's unaligned.
![screenshot_2016-08-23-23-01-22](https://cloud.githubusercontent.com/assets/12042702/17917857/98f11f5c-6985-11e6-919f-2a605f085206.jpg)